### DISCUSSION TITLE
How to update the app using Altstore?

### DISCUSSION THREAD

**Discussion by LeonardoCampanha** [qnblackcat/uYouPlus#663] [Q&A] [answered]
Is it necessary uninstall, download and install new version every time? Doesn't update through the app working?

**Comment by qnblackcat** [comment, marked as answer]
You can’t update directly through the app. You‘ll need to grab the latest ipa, overwrite the current version with the new one. **No need to uninstall the current version**

**Comment by LeoRedin** [comment]
Heya! When I grab the new IPA can I just install it as I would if I didn't have it installed and it will overwrite the one I have?

**Reply by pha1n0q** [reply]
Yes - your user data will be retained.
